doc/api/deprecations.md

```diff
@@ -3380,6 +3380,21 @@ In a future version of Node.js, [`assert.CallTracker`][],
 will be removed.
 Consider using alternatives such as the [`mock`][] helper function.
 
+### DEP0174: the `globalPreload` hook
+
+<!-- YAML
+changes:
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/48842
+    description: Superseded be `initialize`.
+-->
+
+Type: Runtime
+
+In a future version of Node.js, the `globalPreload` hook,
+will be removed. Consider using alternatives such as the
+`initialize` hook.
+
```

Comment:
I'm -1 on deprecating an experimental API. A deprecated API is expected to follow the semver contract, an experimental API is not.
```suggestion
```